agent/agent.go

```diff
@@ -588,12 +595,6 @@ func (a *Agent) runTask(ctx context.Context, tc *taskContext) (shouldExit bool,
 	tc.taskConfig.WorkDir = tc.taskDirectory
 	tc.taskConfig.Expansions.Put("workdir", tc.taskConfig.WorkDir)
 
-	grip.Info(message.Fields{
-		"message":     "running task",
-		"task_id":     tc.task.ID,
-		"task_secret": tc.task.Secret,
-	})
-
```

Comment:
why is this being removed?

Reply:
I removed it because it didn't seem like it served any purpose and it logs the task secret, which we should avoid logging if possible.

Reply:
We can just remove the secret part. I think it could be helpful when debugging things.

Reply:
Sure, I re-added without the secret.